jdbc-v2/src/main/java/com/clickhouse/jdbc/ConnectionImpl.java

```diff
@@ -118,14 +118,22 @@ public PreparedStatement prepareStatement(String sql) throws SQLException {
     @Override
     public CallableStatement prepareCall(String sql) throws SQLException {
         checkOpen();
-        throw new SQLFeatureNotSupportedException("CallableStatement not supported", ExceptionUtils.SQL_STATE_FEATURE_NOT_SUPPORTED);
+        if (!config.isIgnoreUnsupportedRequests()) {
```

Comment:
I would think this is totally valid exception because if this is called then NPE will be thrown anyway.
In most cases we can ignore parameters, but not a whole functionality.